content/blog/2020/06/2020-06-28-WinSW-yaml-support.adoc

```diff
@@ -0,0 +1,113 @@
+---
+layout: post
+title: "Windows Service Wrapprt : YAML Configuration Support - GSoC Phase - 01 Updates"
+tags:
+- winsw
+- windows
+- jenkins
+- gsoc
+- gsoc2020
+author: buddhikac96
+description: "This blog post discuss about phase 01 updates of Windows Service Wrapper Yaml support project of GSoC - 2020"
+opengraph:
+  image: /images/gsoc/opengraph.png
+---
+
+
+=== Introduction
+
+On Windows machines, Jenkins server and agents can be installed as Windows Services to get better robustness and manageability within the system. This is a functionality bundled into the Jenkins core directly. When installed as a service, Jenkins uses the Windows Service Wrapper executable (.NET, written in C#) which is being configured by XML config files. Currently, there are only a few configuration checks there (no XML Schema, limited validation, etc.), and it’s often that the service wrapper is misconfigured by Jenkins users.
```

Comment:
`and it’s often that the service wrapper is misconfigured by Jenkins users.` Should be `and it’s often that a Jenkins user misconfigures the service wrapper.`